mergeDatasets.js

```diff
@@ -153,7 +157,9 @@ BSkyLoadRefreshDataframe( "{{selected.out | safe}}" )
                     label: localization.en.by,
                     placeholder: "",
                     allow_spaces:true,
-                    extraction: "TextAsIs",
+                    //extraction: "TextAsIs",
```

Comment:
you better remove then comment - it could result in a lot of commented settings and looks quite unprofessional

Reply:
Done.